src/Entity/FooterLinkGeneral.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\oe_corporate_blocks\Entity;
+
+use Drupal\Core\Config\Entity\ConfigEntityBase;
+use Drupal\Core\Url;
+
+/**
+ * Defines the footer general link configuration entity.
+ *
+ * @ConfigEntityType(
+ *   id = "footer_link_general",
+ *   label = @Translation("Footer Link General"),
+ *   handlers = {
+ *     "view_builder" = "Drupal\Core\Entity\EntityViewBuilder",
+ *     "list_builder" = "Drupal\oe_corporate_blocks\FooterLinkGeneralListBuilder",
+ *     "route_provider" = {
+ *       "html" = "Drupal\Core\Entity\Routing\AdminHtmlRouteProvider",
+ *     },
+ *     "form" = {
+ *       "add" = "Drupal\oe_corporate_blocks\Form\FooterLinkGeneralForm",
+ *       "edit" = "Drupal\oe_corporate_blocks\Form\FooterLinkGeneralForm",
+ *       "delete" = "Drupal\oe_corporate_blocks\Form\FooterLinkGeneralDeleteForm"
+ *     }
+ *   },
+ *   config_prefix = "footer_link.general",
+ *   admin_permission = "administer footer link general",
+ *   entity_keys = {
+ *     "id" = "id",
+ *     "label" = "label",
+ *     "uuid" = "uuid",
+ *     "weight" = "weight"
+ *   },
+ *   config_export = {
+ *     "id",
+ *     "label",
+ *     "url",
+ *     "weight"
+ *   },
+ *   links = {
+ *     "add-form" = "/admin/config/footer-link-general/add",
+ *     "edit-form" = "/admin/config/footer-link-general/{footer_link_general}/edit",
+ *     "delete-form" = "/admin/config/footer-link-general/{footer_link_general}/delete",
+ *     "collection" = "/admin/config/footer-link-general"
```

Comment:
We move this one under structure, we did the same with breadcrumbs on EWCMS. There we used the following:
```
 *     "add-form" = "/admin/structure/breadcrumb_root_item/add",
 *     "edit-form" = "/admin/structure/breadcrumb_root_item/{breadcrumb_root_item}/edit",
 *     "delete-form" = "/admin/structure/breadcrumb_root_item/{breadcrumb_root_item}/delete",
 *     "collection" = "/admin/structure/breadcrumb_root_item"
```

So let's do as we did there and use the entity ID `footer_link_social`:

```
 *     "add-form" = "/admin/structure/footer_link_general/add",
 *     "edit-form" = "/admin/structure/footer_link_general/{footer_link_general}/edit",
 *     "delete-form" = "/admin/structure/footer_link_general/{footer_link_general}/delete",
 *     "collection" = "/admin/structure/footer_link_general"
```

Reply:
Agreed to keep it under `/config` but to still use the entity ID in the path.